.release-it.json

```diff
@@ -11,7 +11,7 @@
     "release": true
   },
   "hooks": {
-    "before:init": ["yarn build", "scripts/remove-workspace.js"],
```

Comment:
🤔 I don't think we need any more of these scripts(`remove-workspace` & `restore-workspace`), @crherman7 could you please confirm that?

Reply:
for now we will need this until we get into our discrete pacakges as the root is private :/

Reply:
I thought that after removing `workspace` from package.json, we could remove it